src/indigo/components/__snapshots__/Check.test.js.snap

```diff
@@ -3,17 +3,17 @@
 exports[`<Check /> Checkbox - checked 1`] = `
 <svg
   aria-hidden="true"
-  className="svg-inline--fa fa-check fa-w-16 "
+  className="svg-inline--fa fa-check "
   data-icon="check"
   data-prefix="fas"
   focusable="false"
   role="img"
   style={Object {}}
-  viewBox="0 0 512 512"
+  viewBox="0 0 448 512"
   xmlns="http://www.w3.org/2000/svg"
 >
   <path
-    d="M173.898 439.404l-166.4-166.4c-9.997-9.997-9.997-26.206 0-36.204l36.203-36.204c9.997-9.998 26.207-9.998 36.204 0L192 312.69 432.095 72.596c9.997-9.997 26.207-9.997 36.204 0l36.203 36.204c9.997 9.997 9.997 26.206 0 36.204l-294.4 294.401c-9.998 9.997-26.207 9.997-36.204-.001z"
+    d="M438.6 105.4C451.1 117.9 451.1 138.1 438.6 150.6L182.6 406.6C170.1 419.1 149.9 419.1 137.4 406.6L9.372 278.6C-3.124 266.1-3.124 245.9 9.372 233.4C21.87 220.9 42.13 220.9 54.63 233.4L159.1 338.7L393.4 105.4C405.9 92.88 426.1 92.88 438.6 105.4H438.6z"
```

Comment:
Jaktože se změnilo tolik ikon (SVG kódu)?

Reply:
Myslím že to trochu ladily no, aby to nebylo že změnily jen verzi a lidi zaplaťe znovu..

Reply:
aha, já myslel že jen přidávali nový..

Reply:
jo to stopro tam jsou i nové a nové effecty apod